app/helpers/application_helper.rb

```diff
@@ -1,2 +1,18 @@
 module ApplicationHelper
+  def bootstrap_class_for flash_type
+    { success: "alert-success", error: "alert-danger", alert: "alert-warning", notice: "alert-info" }[flash_type.to_sym] || flash_type.to_s
+  end
+
+  def flash_messages(opts = {})
+    flash.each do |msg_type, message|
+      concat(content_tag(:div, message, class: "alert #{bootstrap_class_for(msg_type)} alert-dismissible", role: 'alert') do
+        concat(content_tag(:button, class: 'close', data: { dismiss: 'alert' }) do
+          concat content_tag(:span, '&times;'.html_safe, 'aria-hidden' => true)
```

Comment:
Oh, I thought you had mentioned that you need to re-do some tests. If everything is passing and works in dev, then definitely we can merge. Although we should merge mine in first since it's smaller (and is a lower numbered PR).

On commit messages, of course it's going to be subjective, but a convention is to keep them shorter. If people like Sean prefer them longer, that is a personal opinion and not the standard. I would rather the person be able to wrap up what they are doing in a single line of code, because the point of the code review is to go over anything that looks strange.

I think it's fine if you keep them lengthy. I usually skip the commit messages and go straight to the code anyway, so it's not as big of a deal as I'm probably making it sound. Mine will probably stay on the shorter side unless you guys would like me to make them longer.